scripts/operations_tests/operations_tests.py

```diff
@@ -67,6 +67,16 @@ def __bool__(self):
         return self.status == "pass"
 
 
+def process_params(param):
+    """
+    Handle parameter values that cannot be encoded natively in json
+    """
+    if isinstance(param, list):
+        if param[0] == "tuple":
+            return tuple(param[1:])
```

Comment:
Add a ValueError message something like not supported format, first element needs to be "tuple"

Reply:
This is not needed here, because we want to fall through and return the original if the first item is not `tuple`.